support/k8s/k8s-workload-registrar/mode-crd/config/spire-server-registrar.yaml

```diff
@@ -212,7 +212,7 @@ spec:
       shareProcessNamespace: true
       containers:
         - name: spire-server
-          image: gcr.io/spiffe-io/spire-server:1.3.0
```

Comment:
We're currently in the process of changing all [gcr references over to ghcr](https://github.com/spiffe/spire/issues/3545) - it would be nice to pull these up as well since we're in here bumping it

Reply:
When i switch over the ghcr images, spire agent gives me these errors. Whats different between the gcr and ghcr images?

```
time="2022-12-28T04:37:41Z" level=info msg="Starting Workload and SDS APIs" address=/run/spire/sockets/agent.sock network=unix subsystem_name=endpoints
time="2022-12-28T04:37:41Z" level=warning msg="Connection failed during accept" error="could not read caller stat: open /proc/18182/stat: no such file or directory" subsystem_name=endpoints
time="2022-12-28T04:37:42Z" level=warning msg="Connection failed during accept" error="could not read caller stat: open /proc/18182/stat: no such file or directory" subsystem_name=endpoints
time="2022-12-28T04:37:43Z" level=warning msg="Connection failed during accept" error="could not read caller stat: open /proc/18182/stat: no such file or directory" subsystem_name=endpoints
```

Reply:
I also see this on my M1 Mac.

Reply:
Opened https://github.com/spiffe/spire/issues/3749 to track this. Suggest to stick with non-scratch images until we can figure out whats going with the scratch images on ARM platforms.

Reply:
Thanks for opening that issue @faisal-memon , I agree we should move forward with gcr until we figure out #3749